extension-androidauto/build.gradle.kts

```diff
@@ -53,6 +53,6 @@ project.apply {
   from("$rootDir/gradle/ktlint.gradle")
   from("$rootDir/gradle/lint.gradle")
   from("$rootDir/gradle/jacoco.gradle")
-//  from("$rootDir/gradle/sdk-registry.gradle")
+  from("$rootDir/gradle/sdk-registry.gradle")
```

Comment:
we haven't decide on the versioning and release schedule yet.

Reply:
Alright removing